docs/routing/route_configuration_options.md

```diff
@@ -0,0 +1,75 @@
+# Route configuration options
```

Comment:
now our `docs/assets` directory looks like a dump) let's create subdirectory `routes` and relocate all related assets